packages/synapse-react-client/src/components/DropdownSelect/DropdownSelect.tsx

```diff
@@ -71,8 +71,14 @@ export default function DropdownSelect(props: DropdownSelectProps) {
 
   return (
     <React.Fragment>
-      <ButtonGroup variant={variant} ref={anchorRef} aria-label="split button">
+      <ButtonGroup
+        variant={variant}
+        ref={anchorRef}
+        aria-label="split button"
+        size="small"
+      >
         <Button
+          className="optionsButton"
```

Comment:
This component is used in other places so the `size` and `className` props should be passed as their own props

Reply:
I suggest not worrying about passing a className to `Button` and just have `DropdownSelectProps` extend `ButtonGroup` props, then forward extra props to the `ButtonGroup` component